blog-service/2023-06-20-metrics.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Enhancements to rate & delta Operators (Metrics)
```

Comment:
```suggestion
title: Enhancements to Rate and Delta Operators (Metrics)
```

Reply:
Thanks @jpipkin1 for the quick review. I have maintained consistency here as we have written the operators' titles everywhere in small caps. Sharing the reference link here. https://help.sumologic.com/docs/metrics/metrics-operators/delta/

Reply:
Let me know what do you suggest now.

Reply:
That sounds fine, @JainM6.